src/ZendCodingStandard/Utils/LicenseUtils.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
+ * @license   https://github.com/zendframework/zend-coding-standard/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\CodingStandard\Utils;
+
+use SplFileInfo;
+
+/**
+ * License Utility class
+ */
+class LicenseUtils
+{
+    public static $copyrightLine = 'Copyright (c) %s, Zend Technologies USA, Inc.';
+
+    public static $copyright = <<<EOF
+Copyright (c) %s, Zend Technologies USA, Inc.
+
+All rights reserved.
+
+EOF;
```

Comment:
I think this can actually be truncated to only the first line. We do not reserve all rights; the `LICENSE.md` grants the user/consumer a bunch of rights, _so long as the copyright is maintained and propagated_.